straxen/plugins/pulse_processing.py

```diff
@@ -5,25 +5,34 @@
 import strax
 import straxen
 
+from straxen.get_corrections import is_cmt_option, test_is_cmt_option
+
 export, __all__ = strax.exporter()
 __all__ += ['NO_PULSE_COUNTS']
 
 # These are also needed in peaklets, since hitfinding is repeated
 HITFINDER_OPTIONS = tuple([
     strax.Option(
-        'hit_min_amplitude',
-        default='pmt_commissioning_initial',
-        help='Minimum hit amplitude in ADC counts above baseline. '
-             'See straxen.hit_min_amplitude for options.'
+    'hit_min_amplitude', track=True,
+    default='pmt_commissioning_initial', 
+    help='Minimum hit amplitude in ADC counts above baseline. '
+            'Specify as a tuple of length n_tpc_pmts, or a number,'
+            'or a string like "pmt_commissioning_initial" which means calling'
+            'hitfinder_thresholds.py'
+            'or a tuple like (correction=str, version=str, nT=boolean),'
+            'which means we are using cmt.'
     )])
 
 HITFINDER_OPTIONS_he = tuple([
     strax.Option(
-        'hit_min_amplitude_he', track=True,
-        default="pmt_commissioning_initial_he",
-        child_option=True, parent_option_name='hit_min_amplitude',
-        help='Minimum hit amplitude in ADC counts above baseline for the high energy channels. '
-             'See straxen.hit_min_amplitude for options.'
+    'hit_min_amplitude_he',
+    default='pmt_commissioning_initial_he', track=True,
+    help='Minimum hit amplitude in ADC counts above baseline. '
+            'Specify as a tuple of length n_tpc_pmts, or a number,'
+            'or a string like "pmt_commissioning_initial" which means calling'
+            'hitfinder_thresholds.py'
+            'or a tuple like (correction=str, version=str, nT=boolean),'
+            'which means we are using cmt.'
     )])
```

Comment:
Thanks @jorana . There are a couple of things here. Let me make a summary here:

1.  Yes we can use the the former settings, but it will not load the processed data immediately in my test. It still has to do some re-computing, because of the discrepancy described in [my previous comment.](https://github.com/XENONnT/straxen/pull/543#issuecomment-860169318) I have no idea for the reason yet. We want to load data immediately when we use the old default right?
2. For now I am making the default still old style because I will fail the auto tests if I use the cmt style defaults. I am not sure how to modify the auto tests to avoid such issue. Maybe you have already mentioned it in[ your previous comments](https://github.com/XENONnT/straxen/pull/543#pullrequestreview-682713171), but I am still not so sure how it works... Would you please confirm?